dom-manipulation/main.ts

```diff
@@ -0,0 +1,25 @@
+const $hotButton = document.querySelector('.hot-button');
+const $clickCount = document.querySelector('.click-count');
+let timesClicked: any = 0;
```

Comment:
What data type is `0`? That's what you should be annotating `timesClicked` as rather than `any`